test/jdk/sun/net/www/http/HttpClient/MultiThreadTest.java

```diff
@@ -305,12 +416,41 @@ public static void incRequests () {
             }
         }
 
+        /**
+         * Tries to figure out whether this worker might have been idle for more
+         * than the KEEP_ALIVE timeout. This will be true if the worker detected
+         * that the idle timeout was exceeded between two consecutive request, or
+         * if the time between the last reply
```

Comment:
this needs cleanup

Reply:
In what way?

Reply:
Oh - I see - I didn't finish that sentence.